### PR TITLE
Migrate ComposedChart to typescript/rtl

### PR DESCRIPTION
**Pull request by turculaurentiu91**
## How Has This Been Tested?
By running `npm run test` on Ubuntu linux with node 16

## Types of changes

- [x] Refactoring

## Checklist:

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.

### CODE REVIEW

File: test-jest/chart/ComposedChart.spec.tsx [hunk 1]
```diff
@@ -52,37 +46,8 @@ describe('<ComposedChart />', () => {
...
-      <ComposedChart width={800} height={400} data={data} margin={{ top: 20, right: 20, bottom: 20, left: 20 }}>
-        <XAxis dataKey="name" />
-        <YAxis />
-        <CartesianGrid stroke="#f5f5f5" />
-        <Area type="monotone" dataKey="amt" fill="#8884d8" stroke="#8884d8" />
-        <Line type="monotone" dataKey="uv" stroke="#ff7300" />
-      </ComposedChart>,
-    );
-
-    expect(wrapper.state().xAxisMap[0].realScaleType).to.equal('point');
```
Comment (turculaurentiu91) [inline test-jest/chart/ComposedChart.spec.tsx:66]: Cannot read the component state directly using rtl, you have to read the dom, but I couldn't find any dom differences between those two tests.

File: test-jest/chart/ComposedChart.spec.tsx [hunk 2]
```diff
@@ -45,60 +46,27 @@ describe('<ComposedChart />', () => {
...
-  //     <ComposedChart width={800} height={400} data={data} margin={{ top: 20, right: 20, bottom: 20, left: 20 }}>
-  //       <XAxis dataKey="name" />
-  //       <YAxis />
-  //       <CartesianGrid stroke="#f5f5f5" />
-  //       <Area type="monotone" dataKey="amt" fill="#8884d8" stroke="#8884d8" />
-  //       <Line type="monotone" dataKey="uv" stroke="#ff7300" />
-  //     </ComposedChart>,
-  //   );
-
-  //   expect(wrapper.state().xAxisMap[0].realScaleType).to.equal('point');
```
Comment (turculaurentiu91) [inline test-jest/chart/ComposedChart.spec.tsx:59]: Cannot read component state directly. With RTL you have to read the DOM, and I couldn't find anything different related to this state.

File: test-jest/chart/ComposedChart.spec.tsx [hunk 3]
```diff
@@ -45,60 +46,27 @@ describe('<ComposedChart />', () => {
...
-  //       <XAxis dataKey="name" />
-  //       <YAxis />
-  //       <CartesianGrid stroke="#f5f5f5" />
-  //       <Area type="monotone" dataKey="amt" fill="#8884d8" stroke="#8884d8" />
-  //       <Bar dataKey="pv" barSize={20} fill="#413ea0" />
-  //       <Line type="monotone" dataKey="uv" stroke="#ff7300" />
-  //     </ComposedChart>,
-  //   );
-
-  //   expect(wrapper.state().xAxisMap[0].realScaleType).to.equal('band');
```
Comment (turculaurentiu91) [inline test-jest/chart/ComposedChart.spec.tsx:74]: Cannot read component state directly. With RTL you have to read the DOM, and I couldn't find anything different related to this state.

**Review by ckifer — APPROVED** [review]
Yeah... we'll have to adjust testing style a little bit to align better with RTL. Tough with svgs but this is a good start. Thanks

**Pull request merged**